andela/travel_tool_back
# #160370625 Implement edit comment functionality

#### What does this PR do?
This PR implements the edit comment functionality which enables a user to edit own comment if errors were made in such comment.

#### Description of Task to be completed?
- Write tests for editing comment endpoint
- Create an endpoint for editing a comment

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Change into the project directory -` cd travel_tool_back`
- Install project dependencies using `yarn install`
- Run database migrations using `yarn run db:migrate`
- Seed the database using `yarn run db:seed`
- Start the app using make start
- Navigate to [travela](http://travela-staging.andela.com/) in your browser and login
- Open the developer tools, locate applications tab, copy `jwt-token` in the cookies
- Check `.env.example` and set up your `.env` appropriately
- Get back to your command line, Install dependencies and start the server - `make start`
- In`Postman`, use the `jwt-token` to send a request by adding a `Header`, key `Authorization` and value the `token`.
- In Postman, in the `x-www-form-urlencoded` body options;
    - [ ] Create a new request
    - [ ] Add a new comment to the request created
- You should be able to update/edit the comment on `localhost:5000/api/v1/comments/:commentId`
- The response should include the edited comment

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160370625](https://www.pivotaltracker.com/story/show/160370625)

#### Screenshots (if appropriate)
<img width="1277" alt="screen shot 2018-09-14 at 15 07 02" src="https://user-images.githubusercontent.com/12033494/45553689-3b2b2800-b83d-11e8-9243-fe0c7bd95fd5.png">





#### Questions:
None
